src/niscope/metadata/enums_addon.py

```diff
@@ -827,4 +827,197 @@
             },
         ],
     },
+    'ArrayMeasurement': {
+        'values': [
```

Comment:
Why isn't this in the extracted metadata?

Reply:
Because it is for functions only. CVI .sub files are only for attributes.

Reply:
Got it.
If you have a chance, you should give https://github.com/ni/nimi-python/wiki/API-Metadata a pass to get it to be more up to date. It needs some love.